pages/requests/create-renewal.tsx

```diff
@@ -0,0 +1,208 @@
+import Layout from '@components/admin/Layout'; // Layout component
+import { Text, Box, Flex, Stack, Button, GridItem, Input } from '@chakra-ui/react'; // Chakra UI
+import { useState } from 'react'; // React
+import PermitHolderInformationForm from '@components/admin/requests/forms/PermitHolderInformationForm'; //Permit holder information form
+import { PermitHolderInformation } from '@tools/components/admin/requests/forms/types'; //Permit holder information type
+import DoctorInformationForm from '@components/admin/requests/forms/DoctorInformationForm'; //Doctor information form
+import { DoctorInformation } from '@tools/components/admin/requests/forms/doctor-information-form'; //Doctor information type
+import AdditionalQuestionsForm from '@components/admin/requests/forms/renewals/AdditionalQuestionsForm'; //Additional questions form
+import { AdditionalQuestions } from '@tools/components/admin/requests/forms/types'; //Additional questions type
+import PaymentDetailsForm from '@components/admin/requests/forms/PaymentDetailsForm'; //Payment details form
+import { PaymentDetails } from '@tools/components/admin/requests/forms/types'; //Payment details type
+import { PaymentType, Province } from '@lib/graphql/types'; //GraphQL types
+
+export default function CreateRenewal() {
+  const [permitHolderID] = useState('303240');
+  const [permitHolderInformation, setPermitHolderInformation] = useState<PermitHolderInformation>({
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    postalCode: '',
+  });
+  const [doctorInformation, setDoctorInformation] = useState<DoctorInformation>({
+    phone: '',
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    postalCode: '',
+    name: '',
+    mspNumber: 0,
+  });
+  const [additionalQuestions, setAdditionalQuestions] = useState<AdditionalQuestions>({
+    usesAccessibleConvertedVan: false,
+    requiresWiderParkingSpace: false,
+  });
+  const [paymentDetails, setPaymentDetails] = useState<PaymentDetails>({
+    paymentMethod: PaymentType.Mastercard,
+    donationAmount: 25,
+    shippingAddressSameAsHomeAddress: true,
+    shippingFullName: '',
+    shippingAddressLine1: '',
+    shippingAddressLine2: '',
+    shippingCity: '',
+    shippingProvince: Province.Bc,
+    shippingPostalCode: '',
+    billingAddressSameAsHomeAddress: true,
+    billingFullName: '',
+    billingAddressLine1: '',
+    billingAddressLine2: '',
+    billingCity: '',
+    billingProvince: Province.Bc,
+    billingPostalCode: '',
+  });
+
+  return (
+    <Layout footer={false}>
+      <GridItem display="flex" flexDirection="column" colSpan={12} paddingX="108px">
+        <Flex>
+          <Text textStyle="display-large">
+            {`New Renewal Request (User ID:`}
+            <Box as="span" color="primary">
+              {' '}
+              {permitHolderID}
+            </Box>
```

Comment:
This should be a Next.js `Link` component that links to `/permit-holder/<permitHolderID>`. There are examples of this around the app